roles/kafka_broker/tasks/update_metadata.yml

```
@@ -0,0 +1,64 @@
+---
+- name: Get current Kafka features
+  ansible.builtin.command: >-
+    /usr/bin/kafka-features
+    --bootstrap-server {{ hostvars[inventory_hostname]|confluent.platform.resolve_hostname }}:{{kafka_broker_listeners[kafka_broker_inter_broker_listener_name]['port']}}
+    --command-config {{ kafka_broker.client_config_file }}
+    describe
+  register: current_features
+  failed_when: false
+  changed_when: false
+  check_mode: false
+  when: kafka_broker_metadata_version is defined
+
+- name: Get Kafka feature metadata.version
+  ansible.builtin.set_fact:
+    metadata_version_enabled: "{{ 'metadata.version' in current_features.stdout }}"
+    metadata_versions: >-
+      {{
+          dict(current_features.stdout
+            | regex_search('Feature: metadata.version.*$')
```

```suggestion
            | regex_search('Feature: metadata.version.*$', multiline=true)
```

Needed for CP 8, where the line we are looking for is no longer the last line in the output.